ai.html

```diff
@@ -0,0 +1,600 @@
+                                    >
+                                        <path
+                                            d="M7 7C5.3 7 4 8.3 4 10L4 13L46 13L46 10C46 8.3 44.7 7 43 7L7 7 z M 8 9C8.552 9 9 9.448 9 10C9 10.552 8.552 11 8 11C7.448 11 7 10.552 7 10C7 9.448 7.448 9 8 9 z M 11 9C11.552 9 12 9.448 12 10C12 10.552 11.552 11 11 11C10.448 11 10 10.552 10 10C10 9.448 10.448 9 11 9 z M 14 9C14.552 9 15 9.448 15 10C15 10.552 14.552 11 14 11C13.448 11 13 10.552 13 10C13 9.448 13.448 9 14 9 z M 4 15L4 40C4 41.657 5.343 43 7 43L43 43C44.657 43 46 41.657 46 40L46 15L4 15 z M 37.1875 20.019531C37.315281 20.043859 37.441141 20.092922 37.556641 20.169922C38.015641 20.473922 38.139031 21.095688 37.832031 21.554688L33.832031 27.554688C33.802031 27.599688 33.765469 27.635828 33.730469 27.673828C33.720469 27.683828 33.717031 27.697031 33.707031 27.707031C33.702031 27.712031 33.696406 27.71375 33.691406 27.71875C33.606406 27.80075 33.507344 27.862203 33.402344 27.908203C33.379344 27.918203 33.357984 27.927547 33.333984 27.935547C33.226984 27.973547 33.115953 28 33.001953 28L33 28L32.998047 28C32.885047 28 32.773062 27.974547 32.664062 27.935547C32.630063 27.923547 32.600359 27.906625 32.568359 27.890625C32.527359 27.870625 32.484312 27.857031 32.445312 27.832031C32.416312 27.813031 32.402953 27.784672 32.376953 27.763672C32.350953 27.742672 32.318922 27.732031 32.294922 27.707031L29.294922 24.707031C28.903922 24.316031 28.903922 23.685922 29.294922 23.294922C29.685922 22.903922 30.316031 22.903922 30.707031 23.294922L32.845703 25.431641L36.169922 20.447266C36.399422 20.103016 36.804156 19.946547 37.1875 20.019531 z M 13 23L24 23C24.553 23 25 23.448 25 24C25 24.552 24.553 25 24 25L13 25C12.447 25 12 24.552 12 24C12 23.448 12.447 23 13 23 z M 37.1875 30.019531C37.315281 30.043891 37.441141 30.093172 37.556641 30.169922C38.015641 30.474922 38.139031 31.094688 37.832031 31.554688L33.832031 37.554688C33.802031 37.599688 33.765469 37.635828 33.730469 37.673828C33.720469 37.683828 33.717031 37.697031 33.707031 37.707031C33.702031 37.712031 33.696406 37.71375 33.691406 37.71875C33.606406 37.80075 33.507344 37.862203 33.402344 37.908203C33.379344 37.918203 33.357984 37.927547 33.333984 37.935547C33.226984 37.973547 33.115953 38 33.001953 38L33 38L32.998047 38C32.885047 38 32.773062 37.974547 32.664062 37.935547C32.630063 37.923547 32.600359 37.906625 32.568359 37.890625C32.527359 37.870625 32.484312 37.857031 32.445312 37.832031C32.416312 37.813031 32.402953 37.784672 32.376953 37.763672C32.350953 37.742672 32.318922 37.732031 32.294922 37.707031L29.294922 34.707031C28.903922 34.316031 28.903922 33.685922 29.294922 33.294922C29.685922 32.903922 30.316031 32.903922 30.707031 33.294922L32.845703 35.431641L36.169922 30.447266C36.399422 30.102266 36.804156 29.946453 37.1875 30.019531 z M 13 33L24 33C24.553 33 25 33.447 25 34C25 34.553 24.553 35 24 35L13 35C12.447 35 12 34.553 12 34C12 33.447 12.447 33 13 33 z"
```

Comment:
This does not look great. Mixing binary data into source code.
Hard to maintain embedded binary data. Why not link to a local or CDN svg icon file with a recognizable name?